.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+trim_trailing_whitespace = true
```

Comment:
Hmm, right now the source code is in an inconsistent state about this... so if someone edits a file with trailing whitespace, they'll generate unrelated changes. I suppose we should just bite the bullet and do a wholesale whitespace fix (there's also hard tabs here and there), but until we do that I'm somewhat uncomfortable about this being the default.

Reply:
I agree. I saw this when adding an example, and I guess any file modified/added would start getting fixes, which makes sense.

Reply:
Actually, let's just go for it as is, we can alter this if it becomes annoying.